### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by srikris**
- Clarify that this works on all platforms a bit more explicitly
- Clarify that one does not require a GPU to use it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,9 +104,8 @@ more details on how to use Turi Create.
 GPU Support
 -----------
 
-By default, `turicreate` takes a dependency on the default installation of
-`mxnet`. To enable GPU support after installation of the `turicreate` package,
-please perform the following steps:
+Turiceate **does not require a GPU**, but certain models can be accelerated by the use of a GPU.
```
Comment (gustavla) [inline README.md]: Turi Create

File: README.md [hunk 2]
```diff
@@ -102,9 +104,8 @@ more details on how to use Turi Create.
 GPU Support
 -----------
 
-By default, `turicreate` takes a dependency on the default installation of
-`mxnet`. To enable GPU support after installation of the `turicreate` package,
-please perform the following steps:
+Turiceate **does not require a GPU**, but certain models can be accelerated by the use of a GPU. 
+Since `turicreate` depends on the default installation of `mxnet`. To enable GPU support after installation of the `turicreate` package, please perform the following steps:
```
Comment (gustavla) [inline README.md]: This is not a complete sentence.

Comment (srikris) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (srikris) [reply README.md]: Done.

File: README.md [hunk 3]
```diff
@@ -57,7 +57,7 @@ You can also work with essential machine learning models, organized into algorit
 System Requirements
 -------------------
 
-Turi create works on **Mac**, **Linux**, and **Windows**.
+Turicreate works on **Mac**, **Linux**, and **Windows**.
```
Comment (gustavla) [inline README.md]: Turi Create

You "fixed" the wrong one.

File: README.md [hunk 4]
```diff
@@ -57,6 +57,8 @@ You can also work with essential machine learning models, organized into algorit
 System Requirements
 -------------------
 
+Turicreate works on **Mac**, **Linux**, and **Windows**.
```
Comment (gustavla) [inline README.md]: Turi Create

**Pull request merged**